cmd/training-operator.v1/main.go

```diff
@@ -74,6 +75,7 @@ func main() {
 		"If set, it only monitors kubeflow jobs in the given namespace.")
 	flag.IntVar(&monitoringPort, "monitoring-port", 9443, "Endpoint port for displaying monitoring metrics. "+
 		"It can be set to \"0\" to disable the metrics serving.")
+	flag.IntVar(&controllerThreads, "controller-threads", 10, "Number of worker threads used by the controller.")
```

Comment:
Can we keep current defaults here(1)?

Reply:
Most users are too "lazy" to change the default values, or they wish the default values can handle most situations, I think the default value set to 10 is better?

Reply:
This value is highly dependent on deployment environment and user needs.  I would recommend to keep it default 1 and user can override it based on their needs. (Ref: https://docs.okd.io/latest/operators/operator_sdk/golang/osdk-golang-tutorial.html#osdk-golang-controller-configs_osdk-golang-tutorial)

Reply:
@HeGaoYuan I agree with @johnugeorge. We should keep it default `1` since it would be best to set values as the minimal requirement one as default.

Reply:
Welcome more programmers to convince me of 1 is better than 10. 😁😁

Reply:
I was totally agree that the default value should be 1 here, regards as following reasons and more,
* the default value of `controller.Options.MaxConcurrentReconciles` is 1, [cf](https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/controller/controller.go#L39)
* if we do not make sure that EVERYTHING now and future are completely thread safe, it's better to keep the minimal working configuration as default. 

Anyway, keep default as DEFAULT would be better in my personal point of view.

Reply:
@HeGaoYuan  Can you update the PR?

Reply:
> @HeGaoYuan Can you update the PR?

ok